<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingForm</class>
 <widget class="QDialog" name="MappingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>507</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
    <item row="1" column="2">
     <widget class="QLineEdit" name="fctIDRepLdt">
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="fctCriticalityRepLdt"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="fctUsageRepLdt"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="fctCriticalitySpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Functionality Channel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="fctDescriptionSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="fctNameRepLdt"/>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="fctChannelSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="fctIDSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Replacement text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Import column</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapped field</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="fctNameSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Functionality Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Functionality Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Functionality ID</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Functionality Usage</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Functionality Criticality</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="fctDescriptionRepLdt"/>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="fctUsageSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="fctChannelRepLdt"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="fctIDHeaderLdt"/>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="fctNameHeaderLdt"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="fctChannelHeaderLdt"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="fctDescriptionHeaderLdt"/>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="fctCriticalityHeaderLdt"/>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="fctUsageHeaderLdt"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_33">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Header text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Functionality Mapping</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>496</width>
     <height>462</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="8" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Require Usecase</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="ucIDSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Replacement text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Test Type</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Post Condition</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapped field</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Critical Path</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLineEdit" name="ucAutomatedRepLdt"/>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="ucAutomatableRepLdt"/>
    </item>
    <item row="14" column="2">
     <widget class="QLineEdit" name="ucReleaseRepLdt"/>
    </item>
    <item row="15" column="2">
     <widget class="QLineEdit" name="ucFunctionalityRepLdt"/>
    </item>
    <item row="16" column="2">
     <widget class="QLineEdit" name="ucAutomatonRepLdt"/>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="ucAverageSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="ucCriticalSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="ucAutomatedSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="ucAutomatableSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSpinBox" name="ucReleaseSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QSpinBox" name="ucAutomatonSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="ucIDRepLdt"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="ucNameRepLdt"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="ucDescriptionRepLdt"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="ucPreRepLdt"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="ucStepsRepLdt"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="ucPostRepLdt"/>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="ucExcpRepLdt"/>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="ucRequireRepLdt"/>
    </item>
    <item row="11" column="2">
     <widget class="QLineEdit" name="ucAverageRepLdt"/>
    </item>
    <item row="9" column="2">
     <widget class="QLineEdit" name="ucTypeRepLdt"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Average Run Time</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Automated</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Automatable</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Release</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Functionality link</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="ucNameSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Automaton link</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="ucCriticalRepLdt"/>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="ucDescriptionSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="ucPreSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="ucStepsSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="ucPostSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="ucExcpSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="ucRequireSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="ucTypeSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Import column</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Pre Requisite</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Steps</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Short Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Exceptional Behaviour</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_34">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Header text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="ucIDHeaderLdt"/>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="ucNameHeaderLdt"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="ucDescriptionHeaderLdt"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="ucPreHeaderLdt"/>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="ucStepsHeaderLdt"/>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="ucPostHeaderLdt"/>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="ucExcpHeaderLdt"/>
    </item>
    <item row="8" column="3">
     <widget class="QLineEdit" name="ucRequireHeaderLdt"/>
    </item>
    <item row="10" column="3">
     <widget class="QLineEdit" name="ucCriticalHeaderLdt"/>
    </item>
    <item row="9" column="3">
     <widget class="QLineEdit" name="ucTypeHeaderLdt"/>
    </item>
    <item row="11" column="3">
     <widget class="QLineEdit" name="ucAverageHeaderLdt"/>
    </item>
    <item row="12" column="3">
     <widget class="QLineEdit" name="ucAutomatedHeaderLdt"/>
    </item>
    <item row="13" column="3">
     <widget class="QLineEdit" name="ucAutomatableHeaderLdt"/>
    </item>
    <item row="14" column="3">
     <widget class="QLineEdit" name="ucReleaseHeaderLdt"/>
    </item>
    <item row="15" column="3">
     <widget class="QLineEdit" name="ucFunctionalityHeaderLdt"/>
    </item>
    <item row="16" column="3">
     <widget class="QLineEdit" name="ucAutomatonHeaderLdt"/>
    </item>
    <item row="15" column="1">
     <widget class="QSpinBox" name="ucFunctionalitySpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Use case Mapping</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Header row</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>First data row</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="headerRowSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="firstDataRowSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>491</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Information&lt;/h1&gt;
&lt;h2&gt;Purpose&lt;/h2&gt;
This aim to map the excel spreadsheet column with the application data field.
&lt;h2&gt;General&lt;/h2&gt;
&lt;ul&gt;
&lt;li&gt;Column start at 1&lt;/li&gt;
&lt;li&gt;Replacement text : used &lt;/li&gt;
&lt;/ul&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
